Stateless/Properties/AssemblyInfo.cs

```diff
@@ -28,6 +28,6 @@
 [assembly: AssemblyVersion("2.4.0.1")]
 [assembly: AssemblyFileVersion("2.4.0.1")]
 
-[assembly: InternalsVisibleTo("Stateless.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010077d5f127d9877b3b813017ddefb168b265f8041ec908984bf36071156ff911a7a27b5243decac72586ecf45edfb93c50fdf063639e39edff2238a2b999caf0b2c9767a1517cb1df747b58853503e03e1fb12f1336c35aa97ad8a8bc2984b7cae9f49cad2eedebb8cf59b3bd5f24fd9f3a53bd1a2d8499f4fe0d938873b0101ca")]

```

Comment:
`[InternalsVisibleTo]` has to have a key specified if it appears in a signed library. With Stateless.dll signed, the tests can't get internal access unless they're also signed.